src/fabric/src/fabric_streams.erl

```diff
@@ -95,16 +95,23 @@ start(Workers0, Keypos, StartFun, Replacements, RingOpts) ->
 
 cleanup(Workers) ->
     % Stop the auxiliary cleaner process as we got to the point where cleanup
-    % happesn in the regular fashion so we don't want to send 2x the number kill
-    % messages
+    % happens in the regular fashion and we don't want to send 2x the number
```

Comment:
s/the number kill messages/the number of kill messages/

Reply:
Thanks, @big-r81 !